.github/workflows/ci.yaml

```diff
@@ -65,7 +65,7 @@ jobs:
           which libtool
           # Install socket_vmnet to `_output/bin` which is used in $PATH
           SOCKET_VMNET_TEMP_PREFIX=$(pwd)/_output/ make lima-socket-vmnet
-          make install.lima-dependencies dependencies
+          make dependencies
```

Comment:
`install.lima-dependencies` is a dependency of `dependencies` on macOS.

Maybe an opportunity to have each platform define a Make target of `install.dependencies` which is called by `finch` repository.